RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -243,35 +252,35 @@ private void ApplyAbstractFactoryConvention(IEnumerable<Assembly> assemblies)
                 .Configure(binding => binding.InSingletonScope()));
         }
 
-        // note: InspectionBase implementations are discovered in the Rubberduck assembly via reflection.
-        private void BindCodeInspectionTypes()
+        // note: IInspection implementations are discovered in all assemblies, via reflection.
+        private void BindCodeInspectionTypes(IEnumerable<Assembly> assemblies)
         {
-            var inspections = Assembly.GetExecutingAssembly()
-                                      .GetTypes()
-                                      .Where(type => type.BaseType == typeof (InspectionBase));
+            var inspections = assemblies
+                .SelectMany(a => a.GetTypes().Where(type => type.IsClass && !type.IsAbstract && type.GetInterfaces().Contains(typeof (IInspection))))
+                .ToList();
```

Comment:
This means any .dll under the `\Plug-ins` folder can contain `IInspection` implementations, we'll load them, and bind them.